hbase-server/src/main/java/org/apache/hadoop/hbase/master/snapshot/SnapshotFileCache.java

```diff
@@ -305,6 +313,10 @@ public boolean isStopped() {
     return this.stop;
   }
 
+  protected long getLastModifiedTime() {
```

Comment:
Make it package private and add a VisibleForTesting annotation?

Reply:
Done.